ds701_book/20-Recommender-Systems.qmd

```diff
@@ -404,169 +566,232 @@ The strategy for solving is:
 
 Each of the three steps will reduce the overall error.   So we iterate over them until convergence.
```

Comment:
Each of the three steps reduces the overall error. As a result, we iterate over them until convergence.